pkg/controller/types/group.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2019 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package types
+
+import (
+	"github.com/vmware-tanzu/antrea/pkg/apis/controlplane"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+type GroupType string
+
+const (
+	ClusterGroup    GroupType = "ClusterGroup"
+	NamespacedGroup GroupType = "NamespacedGroup"
+)
+
+type GroupReference struct {
+	// Type of the Group.
+	Type GroupType
```

Comment:
Could we use Namespace field value to judge the type?

Reply:
we could.. i followed what we did for NetworkPolicy, but for this internal type, I think we can switch to your suggestion .. will update.

Reply:
removed